driver/ppm_compat_unistd_32.h

```diff
@@ -355,6 +355,7 @@
 #define __NR_ia32_process_vm_readv	347
 #define __NR_ia32_process_vm_writev	348
 #define __NR_ia32_renameat2 349
+#define __NR_ia32_userfaultfd 350
```

Comment:
After doing this, you need to increase `NR_ia32_syscalls` to `351`

Reply:
Forgot it, right! 

Thank you!